visidata/loaders/pandas_freqtbl.py

```diff
@@ -97,7 +97,7 @@ def reload(self):
         # that operates similarly to pd.cut.
         super().initCols()
 
-        df = self.source.rows.df
```

Comment:
So, does opening a freqtbl row still work to get back to a sheet with references to the original rows?  As in, modifications on a freq slice will reflect back to the original sheet?

Reply:
Yes, changes on a freq slice still update the original.

Reply:
Freq slices 

> 
> 
> So, does opening a freqtbl row still work to get back to a sheet with references to the original rows? As in, modifications on a freq slice will reflect back to the original sheet?

I can't tell yet, with or without these changes I'm getting empty freqtbl slices with pandas (this has already been reported in #801 but I haven't dug into that piece yet).